Command/UpdateClientCommand.php

```diff
@@ -39,29 +39,30 @@ protected function configure(): void
             ->addOption(
                 'redirect-uri',
                 null,
-                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
-                'Sets redirect uri for client. Use this option multiple times to set multiple redirect URIs.',
-                []
+                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
+                'Sets redirect uri for client. Use this option multiple times to set multiple redirect URIs. Use it without value to remove existing values.',
+                [0]
```

Comment:
@v-m-i I have several issues with this approach:
1) this hole `[0]` approach seems kind of hacky, why not just leave `[]` as the default & use `empty`:
![image](https://user-images.githubusercontent.com/2445045/78059641-7bec8280-738a-11ea-8391-25022ed96e1d.png)
2) what if someone runs it like this: `bin/console trikoder:oauth2:update-client 1 --redirect-uri=url --redirect-uri`, they would get:
![image](https://user-images.githubusercontent.com/2445045/78059900-fb7a5180-738a-11ea-9629-3a57da0a3e2d.png)

I'm thinking maybe this would be a better approach:
1) leave `InputOption::VALUE_REQUIRED` & `[]` as the default
2) add `--remove-*` equivalents, eg `--remove-grant-type`

Then if someone has a client with the scopes `s1` & `s2`, they could do: `trikoder:oauth2:update-client 1 --scope=s3 --remove-scope=s1` & would end up with a client that has the scopes `s2` & `s3`. Hint: you could use `array_merge` & `array_diff`.

@spideyfusion @X-Coder264 Any thoughts on this one?

Reply:
Hi @HypeMC 

1. I agree it is hacky, you are right, there is no reason to use `[0]` when I could have used `[]`, I will change it if we chose this "set" approach.

2. This comand sets attributes it has recieved, meaning that after running `trikoder:oauth2:update-client 1 --scope=s3` client `1` would have only `s3` scope, regardless of how many scopes it had before. 

We have two requirements to satisfy here:
    1. User can remove all scopes from client
    2. User can set scopes for client

I think that in order to satisfy first requirement our InputOption must be `VALUE_OPTIONAL` because `trikoder:oauth2:update-client 1 --scope` is the only way to remove all scopes with "set" approach?

Add/Remove approach you have mentioned is something I would like more than this "set" approach. If we would decide to go with that approach, I would suggest that `--scope`, and other methods are prefixed with `add`, like this: `trikoder:oauth2:update-client 1 --add-scope=s3 --remove-scope=s1`